### PR TITLE
Create Pinboard bookmarks source

### PR DESCRIPTION
**Pull request by pstuifzand**

### CODE REVIEW

File: kupfer/plugin/pinboard.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        "type": str,
+        "value": "",
+    },
+)
+
+class PinboardBookmarkSource(Source):
+    def __init__(self):
+        super().__init__(_("Pinboard Bookmarks"))
+        token = __kupfer_settings__["token"]
+        self.pinboard = pinboard.Pinboard(token)
```
Comment (KarolBedkowski) [inline kupfer/plugin/pinboard.py:26]: Token in this place may be not initialized or empty (first run).
self.pineboard should be initialized at first get_items. Maybe even not should be object attribute  as long is only used in get_items

**Review by KarolBedkowski — CHANGES_REQUESTED** [review]
Comment in code.
Please also send PR as one commit (squash and rebase).

File: kupfer/plugin/pinboard.py (hunk 1 above)
Comment (pstuifzand) [reply kupfer/plugin/pinboard.py]: I squashed the commits and look if token is set.

**Comment by KarolBedkowski** [comment]
Thank you. Merged

**Pull request closed without merging**